nic_operator/nic_operator_ci_test.sh

```diff
@@ -322,6 +323,11 @@ function test_secondary_network {
         echo "Error: Couldn't delete $sample_file!"
         return $status
     fi
+
+    multus_configuration
```

Comment:
you could just delete /etc/cni/00-multus.conf so kubelet executes the previous CNI config

Reply:
Actually multus is used before the test, but because we are testing multus, the original multus is deleted from the machine before the test and the network operator one is deployed. So to restore the last state the deployment was at before the test, i think redeploying the multus would be best.

Reply:
OK, can you add a comment saying you are restoring secondary network configuration ? otherwise LGTM